scripts/debug-test.js

```diff
@@ -10,7 +10,7 @@ if (!configPath || !rootPath) {
     'Unable to find jest.config.js relative to currently opened file. Run debug-test from an open source file in a jest enabled project.',
   );
 } else {
-  const jestCli = require.resolve('jest-cli/bin/jest.js');
+  const jestCli = require.resolve('jest-cli/bin/jest');
```

Comment:
Jest 27 adds `exports` to package.json which restricts what node will import. `./bin/jest` is included in `exports` but the version with `.js` isn't. I'm going ahead and switching it in this PR to avoid someone having to debug this again later. 😬